wifi-connect/start.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+export DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
+
+
+while [[ true ]]; do
+  # Choose a condition for running WiFi Connect according to your use case:
+
+  # 1. Is there a default gateway?
+  # ip route | grep default
+
+  # 2. Is there Internet connectivity?
+  # nmcli -t g | grep full
+
+  # 3. Is there Internet connectivity via a google ping?
+  wget --spider http://google.com 2>&1
+
+  # 4. Is there an active WiFi connection?
+  # iwgetid -r
+
+  if [ $? -eq 0 ]; then
+      printf 'Skipping WiFi Connect\n'
+  else
+      printf 'Starting WiFi Connect\n'
+      # Start wifi-connect with SSID "balenaFin", Password "balenaFin" and make it exit if no interaction happens within 10 minutes.
```

Comment:
The comment say SSID == `balenaFin` I assume thats just a hold over from where it was copied from?

Reply:
Ah yes, failure to update all the things - good catch